src/libs/elektra/kdb.c

```diff
@@ -999,6 +999,29 @@ KDB * kdbOpen (const KeySet * contract, Key * errorKey)
 		goto error;
 	}
 
+	// TODO (atmaxinger): improve
+	bool existingError = keyGetMeta (errorKey, "error") != NULL;
+
+	if (!existingError)
+	{
+		// set a dummy value to block errors
+		// any errors that occur will be converted into warnings
+		keySetMeta (errorKey, "error", "blocked");
+	}
```

Comment:
I'd actually do this "error blocking" inside of `initHooks`, the `kdbOpen` function doesn't really care about this.